apps/mobile/src/features/send/send-form/hooks/use-send-form-stx.tsx

```diff
@@ -1,13 +1,17 @@
-import { useGenerateStxTokenTransferUnsignedTransaction } from '@/common/transactions/stacks-transactions.hooks';
 import { useToastContext } from '@/components/toast/toast-context';
-import { useSettings } from '@/store/settings/settings';
+import { useNetworkPreferenceStacksNetwork } from '@/store/settings/settings.read';
```

Comment:
I didn't know we had this 🫨  I'll refactor some of my other code soon. I was using `useSettings` the whole time